src/Components/Recommend/Recommend.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Recommend.scss';
+
+class Recommend extends React.Component {
+  state = {
+    recommendList: [],
+  };
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/main/Recommend.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          recommendList: data,
+        })
+      );
+  }
+
+  render() {
+    const { recommendList } = this.state;
+    const list = recommendList.map(item => {
+      const {
+        img,
+        forign,
+        korean,
+        info,
+        width,
+        height,
+        zIndex,
+        unit,
+        price,
+        grade,
+      } = item;
```

Comment:
```suggestion
    const list = recommendList.map(item => {
```
이 경우에는 구조분해 할당하는 것이 오히려 가독성을 해치기 때문에 없는게 나을 거 같습니다

Reply:
### 리뷰 답장
- 말씀하신대로 생각하니 그렇게 보이네요...! 구조분해 할당 부분 수정하겠습니다!

### 수정 사항
```suggestion
const list = recommendList.map(item => {
      return (
        <li>
          <Link to="">
            <img src={item.img} alt={`${item.forign}_img`} />
            <div className="text">
              <span className="name">{`${item.forign} ${item.korean}`}</span>
              <span className="info">{`${item.info}, ${item.width}x${item.height}x${zIndex} ${unit}`}</span>
              <span className="price">{`₩ ${item.price}`}</span>
              <span className="grade">{'★'.repeat(item.grade)}</span>
            </div>
          </Link>
        </li>
```
- 구조 분해할당 수정하고 아래 부분도 전부 수정하였습니다!